lib/parser/grammar.h

```diff
@@ -178,12 +178,12 @@ constexpr auto namedIntrinsicOperator{
     ".EQV." >> pure(DefinedOperator::IntrinsicOperator::EQV) ||
     ".NEQV." >> pure(DefinedOperator::IntrinsicOperator::NEQV) ||
     extension<LanguageFeature::XOROperator>(
-        ".XOR." >> pure(DefinedOperator::IntrinsicOperator::XOR)) ||
+        ".XOR." >> pure(DefinedOperator::IntrinsicOperator::NEQV)) ||
     extension<LanguageFeature::LogicalAbbreviations>(
         ".N." >> pure(DefinedOperator::IntrinsicOperator::NOT) ||
         ".A." >> pure(DefinedOperator::IntrinsicOperator::AND) ||
         ".O." >> pure(DefinedOperator::IntrinsicOperator::OR) ||
-        ".X." >> pure(DefinedOperator::IntrinsicOperator::XOR))};
+        ".X." >> pure(DefinedOperator::IntrinsicOperator::NEQV))};
```

Comment:
This last line might need to be nested in two `extension<>` calls if you don't want to enable `.X.` unless both features have been enabled.

Reply:
Right -- I've fixed it.